app/Http/Controllers/StreamController.php

```diff
@@ -263,9 +263,12 @@ private function extractJsonObject(string $buffer): ?array
         $openBraces = 0;
         $startFound = false;
         $startPos = 0;
+        $openQuote = false;
 
         for($i = 0; $i < strlen($buffer); $i++) {
             $char = $buffer[$i];
+            if ($char === '"') $openQuote = !$openQuote;
```

Comment:
The fix for this functions (although not fully perfect, because it will not reliably capture escaped quotes. However, the OpeanAI responses should never be piped through here anyway (see my other modification)